utils/swift_build_support/swift_build_support/targets.py

```diff
@@ -404,6 +404,8 @@ def host_target():
         elif system == 'OpenBSD':
             if machine == 'amd64':
                 return StdlibDeploymentTarget.OpenBSD.amd64
+            if machine == 'arm64':
```

Comment:
`elif` would better here, as no reason to check this if the previous check passed.